docs/guides/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Have you tried to run docs? Where this item is displayed in the menu?

Do we need to extend hierarchy in the main docs repo to display this content?

https://github.com/OpenZeppelin/docs/blob/main/src/navigation/ethereum-evm.json

if not, would be good to copy this content to local docs repo and try it out and provide instruction for release and creating PR against docs repo.